data/json/recipes/recipe_medsandchemicals.json

```diff
@@ -2522,5 +2522,55 @@
     [ [ "water", 1 ], [ "water_clean", 1 ] ],
     [ [ "ash", 500 ] ]
   ]
+},
+{
+  "result": "anesthesia",
+  "type": "recipe",
+  "category": "CC_OTHER",
+  "subcategory": "CC_OTHER_MEDICAL",
+  "skill_used": "firstaid",
+  "difficulty": 3,
+  "time": 5000,
+  "book_learn":[ [ "textbook_firstaid", 6 ], [ "emergency_book", 5 ] ],
+  "qualities": [
+    { "id": "SEW", "level": 1 },
+    { "id": "CONTAIN", "level": 1 }
+    ],
+  "components":[
+    [ [ "ether", 2700 ] ],
+    [ [ "jar_3l_glass", 1 ] ],
+    [ [ "hose", 1 ] ],
+    [ [ "rag", 2 ] ],
+    [
+      [ "heroin", 5 ],
+      [ "oxycodone", 10 ],
+      [ "tramadol", 15 ],
+      [ "diazepam", 15 ],
```

Comment:
Diazepam isn't an analgesic.

Reply:
You're right, but it could be used as an induction agent. Personally I'd take it out and keep it consistent, but there's an argument for leaving it in

Reply:
consistent with what? I was under the impression the ether would do the bulk of knocking you out, and the other drug was added as an induction agent to help that.

Reply:
Consistent with the other ingredients. Opiates and benzos work along very different pathways, and on my quick read I didn't find anything about using benzos for ether induction (probably because if you have access to benzodiazepines, you'd be using better anaesthesia). They would probably work, mind you, but if you make it all consistent then later if it gets recoded into being an "ether anaesthesia kit" it's easier to figure out appropriate side effects and things.